webpack.config.js

```diff
@@ -33,6 +33,9 @@ const USE_HOT_MODULE_RELOAD =
 const NO_DEV_SERVER = env.NO_DEV_SERVER;
 const FORCE_WEBPACK_DEV_SERVER = env.FORCE_WEBPACK_DEV_SERVER;
 const IS_CI = !!env.CI || !!env.TRAVIS;
+const NO_TS_FORK = env.NO_TS_FORK === 'true';
```

Comment:
I'd replace this with a simple "truthiness" check as many people expect that (or just set `1` as the value)
```suggestion
const NO_TS_FORK = boolean(env.NO_TS_FORK);
```